rpmio/macro.cc

```diff
@@ -9,6 +9,8 @@
 #include <string>
 #include <vector>
 #include <stack>
+#include <cinttypes>
+#include <algorithm>
```

Comment:
I've been mostly trying to keep the C++ includes to alphabetic order. The pre-existing includes in this file are obviously failing to do so, but lets try to stick to that whenever there's no reason to deviate from it. Mind, that's not documented anywhere, we need to really start collecting the new rules somewhere....